example/lib/main.dart

```diff
@@ -51,12 +56,16 @@ class PhoneFieldView extends StatelessWidget {
         textDirection: useRtl ? TextDirection.rtl : TextDirection.ltr,
         child: PhoneFormField(
           key: inputKey,
+
           controller: controller,
           shouldFormat: shouldFormat && !useRtl,
           autofocus: true,
           autofillHints: const [AutofillHints.telephoneNumber],
           countrySelectorNavigator: selectorNavigator,
           defaultCountry: IsoCode.US,
+          showDialCode: showDialCode,
+          isCountrySelectionEnabled: isCountrySelectionEnabled,
+          showFlagInInput: showFlagInput,
```

Comment:
It seems like `showFlagInInput` is deprecated and does nothing. I wouldn't put that in the example if that's the case.

@cedvdb  note to self: remove shofFlagInInput if unused

Reply:
showFlagInInput
Flag will not display when false

Reply:
okay my bad then

Reply:
Is the problem now resolved or what?

Reply:
yes 😅